src/App.tsx

```diff
@@ -1,8 +1,95 @@
-import React from 'react';
+import React, { FC, useState } from 'react';
 import './App.css';
 
-const App = () => (
-  <h1>Dynamic list of TODOs</h1>
-);
+import {
+  TodoWithUser, loadData, Todo, User,
+} from './utils';
+import { TodoList } from './components/TodoList';
+
+const URL_TODOS = 'https://jsonplaceholder.typicode.com/todos';
+const URL_USERS = 'https://jsonplaceholder.typicode.com/users';
+
+
+const App: FC<{}> = () => {
+  const [isLoading, setIsLoading] = useState<boolean>(false);
+  const [isError, setIsError] = useState<string>('');
+  const [todos, setTodos] = useState<TodoWithUser[]>([]);
+  const clickHandler = () => {
+    setIsLoading(true);
+    Promise.all([
+      loadData<Todo[]>(URL_TODOS),
+      loadData<User[]>(URL_USERS),
```

Comment:
это лучше вынести в отдельные функции и вызывать
```suggestion
      getTodos(),
      getUsers(),
```